billing-service/src/main/java/com/petclinic/billing/businesslayer/BillService.java

```diff
@@ -12,6 +12,8 @@
 public interface BillService {
     Mono<BillResponseDTO> getBillByBillId(String billId);
 
+    //Mono<BillResponseDTO> getTaxedAmountByBillIdAndBillAmount(String billId, Double amount);
+
```

Comment:
Is this left here so that eventually it will be implemented in the service impl? If not I don't think it would be a bad idea to get rid of commented out lines that we aren't using.